### PR TITLE
Bootloader

### PR DESCRIPTION
**Pull request by ncros3**
add functions in Gate app to process bootloader commands

### CODE REVIEW

File: Apps/Gate/TinyJSON/bootloader_ex.c [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+ * Function
+ ******************************************************************************/
+/******************************************************************************
+ * @brief Process node responses and send them to the Host
+ * @param luos message
+ * @return None
+ ******************************************************************************/
+void Bootloader_LuosToJson(msg_t *msg)
+{
+    char boot_json[256]  = "\0";
```
Comment (Salem-Luos) [inline Apps/Gate/TinyJSON/bootloader_ex.c]: 256? why? what is the max size of Json msg?

File: Apps/Gate/TinyJSON/bootloader_ex.c [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+ * @brief Process Host commands and send them to the node
+ * @param luos message
+ * @return None
+ ******************************************************************************/
+void Bootloader_JsonToLuos(container_t *container, char *bin_data, json_t const *bootloader_json)
+{
+    if (json_getType(json_getProperty(bootloader_json, "command")) == JSON_OBJ)
+    {
+        // command type
+        char *cmd[16] = {
```
Comment (Salem-Luos) [inline Apps/Gate/TinyJSON/bootloader_ex.c:79]: enum?

File: Apps/Gate/TinyJSON/bootloader_ex.c [hunk 3]
```diff
@@ -26,9 +27,9 @@
  * @param luos message
  * @return None
  ******************************************************************************/
-void Bootloader_LuosToJson(msg_t *msg)
+void Bootloader_LuosToJson(container_t *container, msg_t *msg)
 {
-    char boot_json[256]  = "\0";
+    char boot_json[512]  = "\0";
```
Comment (Salem-Luos) [inline Apps/Gate/TinyJSON/bootloader_ex.c:32]: should be in the prev commit?

File: Apps/Gate/TinyJSON/bootloader_ex.c [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+/*******************************************************************************
+ * Variables
+ ******************************************************************************/
+
+/*******************************************************************************
+ * Function
+ ******************************************************************************/
+/******************************************************************************
+ * @brief Process node responses and send them to the Host
```
[the remark this answers is not in the document]
Comment (ncros3) [reply Apps/Gate/TinyJSON/bootloader_ex.c:26]: I updated function descriptions.

**Review by Salem-Luos — APPROVED** [review]

**Pull request merged**